Create index.md (#1247)
galaxyproject/galaxy-hub: content/events/2022-04-Misconceptions-Writeathon/index.md

---
title: "Galaxy Misconceptions Writeathon"
date: '2022-04-06'
days: 1
tease: 
continent: GL
location: "Online, Global"
image: 
location_url: 
external_url:
gtn: false
contact: "Hans-Rudolf Hotz"
tags: 
---

> The [GGSC](https://galaxyproject.org/community/steering/) wants to address the many misconceptions about Galaxy which are making their rounds. One of the steps is writing a paper. 

Reviewer: ```suggestion
The [Global Galaxy Steering Committee (GGSC)](https://galaxyproject.org/community/steering/) wants to address the many misconceptions about Galaxy which are making their rounds. One of the steps is writing a paper. 
```

> We will hold a 24 hours writeathon to continue working on a document addressing some of the most common misconceptions.

Reviewer: ```suggestion
We will hold a 24-hour writeathon to continue working on a document addressing some of the most common misconceptions.
```

> Please conatc [me](mailto:hrhotz@googlemail.com) if you want to join.

Reviewer: ```suggestion
Please contact [Hans-Rudolf Hotz](mailto:hrhotz@googlemail.com) if you want to join.
```
